### PR TITLE
Changed get_symbols to accept a collection (list) of expressions

### PR DESCRIPTION
**Pull request by MauriceHendrix**
Changed get_symbols to accept a collection (list) of expressions.
To bring it in line with get_equations_for

### CODE REVIEW

File: cellmlmanip/model.py [hunk 1]
```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
             logger.warning('Variable %s already has type=="%s". Skip setting "%s"',
                            variable.dummy, variable.type, variable_type)
 
-    def get_symbols(self, expr):
-        """Returns the symbols in an expression. Works differently to Sympy atoms() because a
+    def get_symbols(self, expressions):
+        """Returns the symbols in a collectiion of expressions. Works differently to Sympy atoms() because a
```
Comment (skeating) [inline cellmlmanip/model.py]: spelling collection

Comment (MauriceHendrix) [reply cellmlmanip/model.py]: fixed

File: cellmlmanip/model.py [hunk 2]
```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
             logger.warning('Variable %s already has type=="%s". Skip setting "%s"',
                            variable.dummy, variable.type, variable_type)
 
-    def get_symbols(self, expr):
-        """Returns the symbols in an expression. Works differently to Sympy atoms() because a
+    def get_symbols(self, expressions):
+        """Returns the symbols in a collection of expressions. Works differently to Sympy atoms() because a
```
Comment (MichaelClerx) [inline cellmlmanip/model.py]: Please rewrite this a bit clearer. I thought it was saying that Sympy's `atoms()` regarded a derivative as a single symbol, but it seems from the code that you mean the opposite.

Comment (MichaelClerx) [reply cellmlmanip/model.py]: To be honest, don't mention Sympy at all (we're not documenting Sympy, plus what if they change their code?)
Just add a note saying that derivatives count as a single symbol.

File: cellmlmanip/model.py [hunk 3]
```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
...
         Derivative instance is regarded as a single symbol"""
         symbols = set()
```
Comment (MichaelClerx) [inline cellmlmanip/model.py]: Second issue with this method: A sympy derivative is not a symbol

https://github.com/sympy/sympy/blob/master/sympy/core/function.py#L1024

It's an expression!

File: cellmlmanip/model.py [hunk 4]
```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
             logger.warning('Variable %s already has type=="%s". Skip setting "%s"',
                            variable.dummy, variable.type, variable_type)
 
-    def get_symbols(self, expr):
-        """Returns the symbols in an expression. Works differently to Sympy atoms() because a
+    def get_symbols(self, expressions):
```
Comment (MichaelClerx) [inline cellmlmanip/model.py]: Should probably be called something like `get_symbols_for` or `get_symbols_for_expression`. I'd expect `model.get_symbols()` to return the symbols in the model

File: cellmlmanip/model.py (hunk 2 above)
Comment (MauriceHendrix) [reply cellmlmanip/model.py]: I just changed what was changed but I don't mind updating the doc string

File: cellmlmanip/model.py (hunk 3 above)
Comment (MauriceHendrix) [reply cellmlmanip/model.py]: Updated the comment to reflect this

File: cellmlmanip/model.py (hunk 4 above)
Comment (MauriceHendrix) [reply cellmlmanip/model.py]: Makes sense, done

**Pull request merged**